specification/maps/data-plane/Weather/stable/2.0/weather.json

```diff
@@ -0,0 +1,3869 @@
+    "OptionalWeatherBasinId": {
+      "name": "basinId",
+      "type": "string",
+      "description": "Basin identifier",
+      "in": "query",
+      "enum": [
+        "AL",
+        "EP",
+        "SI",
+        "NI",
+        "CP",
+        "NP",
+        "SP"
+      ],
+      "x-ms-enum": {
+        "name": "basinId",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "AL",
+            "description": "Atlantic"
+          },
+          {
+            "value": "EP",
+            "description": "Eastern Pacific"
+          },
+          {
+            "value": "SI",
+            "description": "Southwest Indian Ocean"
+          },
+          {
+            "value": "NI",
+            "description": "North Indian Ocean"
+          },
+          {
+            "value": "CP",
+            "description": "Central Pacific"
+          },
+          {
+            "value": "NP",
+            "description": "Northwest Pacific"
+          },
+          {
+            "value": "SP",
+            "description": "South Pacific"
+          }
+        ]
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "RequiredWeatherGovernmentId": {
+      "name": "govId",
+      "description": "Government storm Id",
+      "x-ms-client-name": "governmentStormId",
+      "type": "integer",
+      "format": "int32",
+      "required": true,
+      "in": "query",
+      "x-ms-parameter-location": "method"
+    },
+    "OptionalWeatherGovernmentId": {
+      "name": "govId",
+      "description": "Government storm Id",
+      "x-ms-client-name": "governmentStormId",
+      "type": "integer",
+      "format": "int32",
+      "in": "query",
+      "x-ms-parameter-location": "method"
+    },
+    "WeatherDetails": {
+      "name": "details",
+      "description": "When true, wind radii summary data is included in the response",
+      "x-ms-client-name": "includeDetails",
+      "in": "query",
+      "type": "boolean",
+      "default": false,
+      "x-ms-parameter-location": "method"
+    },
+    "WeatherRadiiGeometry": {
+      "name": "radiiGeometry",
+      "description": "When true, wind radii summary data and geoJSON details are included in the response",
+      "x-ms-client-name": "includeGeometricDetails",
+      "in": "query",
+      "type": "boolean",
+      "default": false,
+      "x-ms-parameter-location": "method"
+    },
+    "Position": {
+      "name": "query",
+      "in": "query",
+      "x-ms-client-name": "coordinates",
+      "description": "The applicable query specified as a comma separated string composed by latitude followed by longitude e.g. \"47.641268,-122.125679\".",
+      "required": true,
+      "x-ms-parameter-location": "method",
+      "type": "array",
+      "collectionFormat": "csv",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "type": "number",
+        "format": "double"
+      }
+    }
+  },
+  "paths": {
+    "/weather/forecast/hourly/{format}": {
```

Comment:
Q: Could the temporal aspect (hourly, minute, quarter daily, daily, current) be an extensible enum parameter rather a path?

Reply:
Thanks for the suggestion!
This is not the new weather api which is out of scope of this time.